### PR TITLE
fix: broken rendering on React 18

### LINKED ISSUE #643: Remove /Add pannel don't work
## Description

i flow the instuction in demo page of document
## Steps to check or reproduce

see my codesanbox : https://codesandbox.io/s/keen-oskar-l2kit9?file=/src/App.js

### PR DESCRIPTION
**Pull request by WoodNeck**
## Issue
#643 

## Details
This fixes #643 & Flickering behavior in React 18 by changing most of the `init()`'s code to be synchronous.

### CODE REVIEW

**Review by daybrush — APPROVED** [review]
LGTM

**Review by qpitlove — APPROVED** [review]
It would be nice to have a range of supported React versions in Readme.md.

File: src/Flicking.ts [hunk 1]
```diff
@@ -853,16 +853,15 @@ class Flicking extends Component<FlickingEvents> {
...
-
-    // Done initializing & emit ready event
-    this._initialized = true;
-    if (preventEventsBeforeInit) {
-      this.trigger = originalTrigger;
-    }
-    this.trigger(new ComponentEvent(EVENTS.READY));
-
-    return;
+    return renderer.render().then(() => {
```
[the remark this answers is not in the document]
Comment (WoodNeck) [reply src/Flicking.ts:856]: I've changed `resize` to `_initialResize` in the initial rendering.
As `_initialResize` doesn't have `render()` in it, I can say that they're identical (most of them).

**Review by malangfox — APPROVED** [review]
LGTM

**Pull request merged**